src/app/tests/suites/certification/ci-pics-values

```diff
@@ -61,8 +61,8 @@ MANUAL_FLOW_CHANGE=1
 WNCV.S.F00=1
 WNCV.S.F02=1
 WNCV.S.F01=1
-WNCV.S.F04=1
-WNCV.S.F03=1
+WNCV.S.F04=0
```

Comment:
Why removing these PICS ?

Reply:
WNCV.S.F03=0 is correct
but WNCV.S.F04=1 should stay at 1

Reply:
I will check it and update